README.md

```diff
@@ -293,52 +293,37 @@ This saved one line of code, and implicitly prevented invoking `some_func` twice
 ### ▶ Strings can be tricky sometimes
 
 <!-- Example ID: 30f1d3fc-e267-4b30-84ef-4d9e7091ac1a --->
-1\.
-
-```py
->>> a = "some_string"
->>> id(a)
-140420665652016
->>> id("some" + "_" + "string") # Notice that both the ids are same.
-140420665652016
-```
 
-2\.
-```py
->>> a = "wtf"
->>> b = "wtf"
->>> a is b
-True
-
->>> a = "wtf!"
->>> b = "wtf!"
->>> a is b
-False
+1\. Notice that both the ids are same.
 
+```python:snippets/2_tricky_strings.py -s 2 -e 3
+assert id("some_string") == id("some" + "_" + "string")
+assert id("some_string") == id("some_string")
 ```
 
-3\.
+2\. `True` because it is invoked in script. Might be `False` in `python shell` or `ipython`
 
-```py
->>> a, b = "wtf!", "wtf!"
->>> a is b # All versions except 3.7.x
-True
+```python:snippets/2_tricky_strings.py -s 6 -e 12
```

Comment:
`-s 6 -e 12` @nifadyev does this mean starting line 6, ending at 12?

Reply:
Yes, you can read more in script [doc](https://github.com/ippie52/markdown_code_embed?tab=readme-ov-file#additional-details). And we may use more explicit `--start` and `--end`